packages/ui/src/TextArea.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { ChangeEvent, FC, FocusEvent, ReactElement, useRef } from 'react'
+import { v4 as uuid } from 'uuid'
+import cn from 'classnames'
+
+import { DataTestProp } from '@hazelcast/helpers'
+import { Error, errorId } from '../src/Error'
+import { Help } from '../src/Help'
+import { Label } from './Label'
+
+import styles from './TextArea.module.scss'
+
+export type TextAreaCoreProps = {
+  name: string
+  value?: string
+  onBlur?: (e: FocusEvent<HTMLTextAreaElement>) => void
+  onChange: (e: ChangeEvent<HTMLTextAreaElement>) => void
+  error?: string
+}
+
+export type TextAreaExtraProps = {
+  label: string
+  textareaClassName?: string
+  errorClassName?: string
+  resizable?: boolean
```

Comment:
nit: Should we have a story for the non-resizable text area?

Reply:
We already do. 👍